.travis.yml

```diff
@@ -66,7 +66,7 @@ env:
     - BUILD_CONAN_SERVER_IMAGE=1
 
 install:
-  - pip install conan
+  - pip install conan humanfriendly
```

Comment:
What humanfriendly does?

Reply:
outputs size as human readable value instead of big number, e.g. 10 Mb, 5 Gb, etc.